internal/manager/config.go

```diff
@@ -63,21 +64,25 @@ type Config struct {
 	GracefulShutdownTimeout           *time.Duration
 
 	// Kong Proxy configurations
-	APIServerHost               string
-	APIServerQPS                int
-	APIServerBurst              int
-	APIServerCAData             []byte
-	APIServerCertData           []byte
-	APIServerKeyData            []byte
-	MetricsAddr                 string
-	ProbeAddr                   string
-	KongAdminURLs               []string
-	KongAdminSvc                OptionalNamespacedName
-	GatewayDiscoveryDNSStrategy cfgtypes.DNSStrategy
-	KongAdminSvcPortNames       []string
-	ProxySyncSeconds            float32
-	InitCacheSyncDuration       time.Duration
-	ProxyTimeoutSeconds         float32
+	APIServerHost                          string
+	APIServerQPS                           int
+	APIServerBurst                         int
+	APIServerCAData                        []byte
+	APIServerCertData                      []byte
+	APIServerKeyData                       []byte
+	MetricsAddr                            string
+	ProbeAddr                              string
+	KongAdminURLs                          []string
+	KongAdminSvc                           OptionalNamespacedName
+	GatewayDiscoveryDNSStrategy            cfgtypes.DNSStrategy
+	GatewayDiscoveryReadinessCheckInterval time.Duration // REVIEW(naming): the name seems too long. Any idea to shorten it?
```

Comment:
I don't mind this being long as it's descriptive and it's clear what it's used for.